aws/table_aws_redshift_cluster.go

```diff
@@ -270,6 +264,19 @@ func tableAwsRedshiftCluster(_ context.Context) *plugin.Table {
 				Description: "A list of Amazon Virtual Private Cloud (Amazon VPC) security groups that are associated with the cluster. This parameter is returned only if the cluster is in a VPC.",
 				Type:        proto.ColumnType_JSON,
 			},
+			{
+				Name:        "logging_details",
```

Comment:
Should this column be called `logging_status` instead? The API call is `DescribeLoggingStatus, so it seems more intuitive to follow with the column name.

Reply:
updated